packages/@azure/servicebus/data-plane/test/streamingReceiver.spec.ts

```diff
@@ -371,36 +371,36 @@ describe("Abandon message", function(): void {
     await testPeekMsgsLength(deadLetterClient, 0);
   }
 
-  it("Partitioned Queue: Multiple abandons until maxDeliveryCount.", async function(): Promise<
+  it("Partitioned Queue: Multiple abandons until maxDeliveryCount, abandon() retains message with incremented deliveryCount.", async function(): Promise<
```

Comment:
This change is not needed.

`Multiple abandons until maxDeliveryCount` makes it clear that we are using abandon and that delivery count gets incremented till maxDeliveryCount.

`abandon() retains message` is misleading. Because the message is not retained in the end, it goes to the dead letter queue